services/alertmanager/alertmanager.go

```diff
@@ -181,6 +195,68 @@ func (svc *Service) updateConfiguration(ctx context.Context) {
 	svc.l.Infof("%s created", alertmanagerConfigPath)
 }
 
+// collectRuleTemplates collects IA rule templates from various sources like
+// templates shipped with PMM and defined by the users.
+func (svc *Service) collectRuleTemplates() {
```

Comment:
Let's keep Alertmanager Service for interaction with Alertmanager only. Our rule templates are not related to it.

Let's move it to Templates service after https://github.com/percona/pmm-managed/pull/527.